obfuscator_bench_test.go

```diff
@@ -0,0 +1,102 @@
+package sqllexer
+
+import (
+	"fmt"
+	"strconv"
+	"testing"
+)
+
+// Benchmark the Tokenizer using a SQL statement
```

Comment:
is it possible to write a similar benchmark for the current parsing code & compare?